tests/test_core.py

```diff
@@ -125,3 +145,9 @@ def test_dataset_example():
         }
     )
     ds_schema.validate(ds)
+
+def test_validate():
```

Comment:
I see what you're going for but I think we need a bit more API to make it happen. What if you instead wrote

```python
schema = DataArraySchema(chunks=False)
```

Whereas `chunks=None` currently indicates that the schema doesn't specify any kind of chunking, `chunks=False` would specify the DataArray be "unchunked".  Conversely, `chunks=True` would require the array to be chunked but otherwise not care about the chunk layout.

Reply:
This is interesting. I am noodling. 🍝

Reply:
Now we allow for chunks to either be a `bool`, a `dict`, or `None`. The case of `chunks=False` gets checked in `test_validate`, and the rest get checked in `test_dataarray_validate_chunks`.